browser-test/src/security.test.ts

```diff
@@ -10,13 +10,17 @@ describe('applicant security', () => {
 
   it('applicant cannot access another applicant data', async () => {
     const {page} = ctx
+    ctx.browserErrorWatcher.ignoreErrorsFromUrl(/.*applicants\/1234\/programs/)
```

Comment:
It looks like you're trying to piggy back your new test on an existing one.  We should have clear focused testing whenever viable, and not use side effects of other tests which are hard to read and understand later.

Seemingly you should have a browser_error_watcher.test.ts.  It can duplicate a test like this but should otherwise be focused on the error watcher not something else.

Reply:
It's not trying to piggyback. This test checks that request returns 401 when user has no access. 401 is considered an error from point of view of BrowserErrorWatcher. So we need to exclude that page from being watched.

Reply:
Ah, Please add a comment to this setup then that this is knowingly ignoring an error that we are generating.